packages/radix-ui-themes/src/components/context-menu.tsx

```diff
@@ -12,21 +12,23 @@ import {
   contextMenuRadioItemPropDefs,
 } from './context-menu.props.js';
 import { extractProps, requireReactElement } from '../helpers/index.js';
-import { Theme, useThemeContext } from '../theme.js';
-import { ThickCheckIcon, ThickChevronRightIcon } from '../icons.js';
+import { Theme, useThemeContext } from './theme.js';
+import { ThickCheckIcon, ThickChevronRightIcon } from './icons.js';
 
-import type { PropsWithoutRefOrColor, GetPropDefTypes } from '../helpers/index.js';
+import type { ComponentPropsWithoutColor } from '../helpers/index.js';
+import type { GetPropDefTypes } from '../props/index.js';
 import { baseMenuContentPropDefs } from './base-menu.props.js';
```

Comment:
weird, we shouldn't need to import the base right as we re-export from it? Same with dropdown menu

Reply:
Oh right, updated